app/src/test/kotlin/com/intive/picover/images/repository/ImagesRepositoryTest.kt

```diff
@@ -36,15 +34,16 @@ class ImagesRepositoryTest : ShouldSpec(
 
 		should("downloaded URIs WHEN storage reference and uri is fetched successfully") {
 			runTest(testDispatcher) {
-				val listResult: ListResult = mockk {
-					every { items } returns listOf(storageItem)
-				}
 				val uriResult: Uri = mockk()
+
```

Comment:
Why it's separated? 
`uriResult` declaration and mocked behavior of `imageReference` in line 39 belongs to "given" block. According to give-when-then approach.